src/pages/cb-bank-accounts/forms/CbBankAccountsForm.js

```diff
@@ -0,0 +1,239 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { CashBankRepository } from 'src/repositories/CashBankRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { useForm } from 'src/hooks/form'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { DataSets } from 'src/resources/DataSets'
+import { FinancialRepository } from 'src/repositories/FinancialRepository'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+
+export default function CbBankAccountsForm({ labels, maxAccess, recordId, invalidate }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const editMode = !!recordId
+
+  const { formik } = useForm({
+    initialValues: {
+      name: null,
+      accountNo: null,
+      currencyId: null,
+      activeStatus: null,
+      groupId: null,
+      accountId: null,
+      accountName: null,
+      IBAN: null,
+      bankId: null,
+      type: 1,
+      bankName: null
+    },
+    maxAccess: maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      name: yup.string().required(' '),
+      accountNo: yup.string().required(' '),
+      currencyId: yup.string().required(' '),
+      activeStatus: yup.string().required(' '),
+      bankId: yup.string().required(' ')
+    }),
+    onSubmit: async obj => {
+      const recordId = obj.recordId
+
+      const response = await postRequest({
+        extension: CashBankRepository.CbBankAccounts.set,
+        record: JSON.stringify(obj)
+      })
+      if (!recordId) {
```

Comment:
You can use directly !obj.recordId , there's no need to create (const recordId=obj.recordId)